src/qt/bitcoin.cpp

```diff
@@ -253,6 +254,23 @@ bool BitcoinApplication::baseInitialize()
     return m_node.baseInitialize();
 }
 
+void BitcoinApplication::registerMetaTypes()
```

Comment:
Why a member function?
Could it be a non-member function, either in unnamed namespace or `static`?

Reply:
Good question 🤷  updated.